LLM_GATEWAY.md

```
@@ -0,0 +1,108 @@
+const llmClient = getDefaultLLMProvider();
+
+// Use the LLM
+const response = await llmClient.chat([
+  {
+    role: 'user',
+    content: 'Your prompt here',
+  }
+]);
+
+console.log(response.text);
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)